src/pages/editor/PublishStrategySettings.tsx

```diff
@@ -58,6 +59,9 @@ export const PublishStrategySettings = () => {
       <Helmet>
         <title id="publish-strategy-label">{t('editor.publish_strategy.publish_strategy')}</title>
       </Helmet>
+      <Typography component="h1" sx={visuallyHidden}>
```

Comment:
Siden alle andre sider ser ut til å vise sin h1 er det kanskje mest naturlig her også? Gir mening for meg at "Publiseringsstrategi" er h1 og "Rettighetspolitikk" blir en h2 under der
```suggestion
      <Typography component="h1">
```